examples/door-lock-app/efr32/include/LockManager.h

```diff
@@ -0,0 +1,149 @@
+/*
+ *
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+#include <app/clusters/door-lock-server/door-lock-server.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "AppEvent.h"
+
+#include "FreeRTOS.h"
+#include "timers.h" // provides FreeRTOS timer support
+
+#include <lib/core/CHIPError.h>
+
+using namespace ::chip;
+
+static constexpr size_t DOOR_LOCK_CREDENTIAL_INFO_MAX_DATA_SIZE = 20;
+
+
+struct LockUserInfo;
+struct LockCredentialInfo;
+struct WeekDaysScheduleInfo;
+struct YearDayScheduleInfo;
+
+struct LockUserInfo
```

Comment:
LockUserInfo and LockCredentialInfo were duplicates and I removed them. 

They are not currently persisted through a reboot, this will fucntionality will need to be added.